ci/consts/versions.go

```diff
@@ -9,19 +9,21 @@ const (
 )
 
 const (
-	GolangVersion = "1.22.3"
+	GolangVersion = distconsts.GolangVersion
+	GolangImage   = distconsts.GolangImage
 	// GolangVersionRuncHack needs to be 1.21, since 1.22 is not yet
 	// supported, and can cause crashes: opencontainers/runc#4233
-	GolangVersionRuncHack = "1.21.7"
+	GolangVersionRuncHackVersion = "1.21.7"
```

Comment:
Should we bump this to `1.21.11` so that we benefit from the latest security fixes?